src/config/commandOptions.js

```diff
@@ -112,4 +112,7 @@ export default {
   dockerNetwork: {
     usage: 'The network that the Docker container will connect to',
   },
+  ignorePackageArtifact: {
+    usage: 'Ignores package artifact config in the serverless.yml when running',
+  },
```

Comment:
This item should have a `type` property as per #1211:

```suggestion
    type: "boolean"`,
  },
```